captum/attr/_core/layer/layer_lrp.py

```diff
@@ -41,6 +41,8 @@ def __init__(self, model, layer) -> None:
         """
         LayerAttribution.__init__(self, model, layer)
         LRP.__init__(self, model)
+        if hasattr(self.model, "device_ids"):
```

Comment:
device_ids is needed to combine the split outputs / gradients in DataParallel models. I think we currently include device_ids as a parameter in any layer attribution method which takes a forward function rather than a model (all except LayerDeepLift). If the DataParallel model is provided itself, then we can just get the device_ids from it; it only needs to be passed when a wrapper forward function is used, which isn't feasible with LayerDeepLift and LRP. I think LayerDeepLift does make use of device_ids internally in the same way through utility methods, but since LRP needs this more frequently, I've just included it as a direct instance variable.

Reply:
sounds good, it looks like only LayerDeepLift doesn't have it and it takes model not forward function.